app/src/test/java/com/android/sample/feature/camera/CameraUtilsTest.kt

```diff
@@ -109,4 +117,54 @@ class CameraUtilsTest {
       }
     }
   }
+
+  @Test
+  fun testUriToBitmapWork() {
```

Comment:
1. **`testUriToBitmapWork`**: Nice coverage here, especially with the pixel-by-pixel comparison! To speed up the test, consider adding a short-circuit check for width and height before running the pixel comparison in `bitmapsAreEqual`. It’ll save time if dimensions don’t match.

2. **`testUriToBitmapErrorHandling`**: Great addition to ensure error handling.